pkg/rc/replication_controller.go

```diff
@@ -184,7 +184,7 @@ func (rc *replicationController) eligibleNodes() ([]string, error) {
 }
 
 func (rc *replicationController) CurrentNodes() ([]string, error) {
-	selector := rc.PodLabels.AsSelector()
+	selector := labels.Everything().Add(rcIdLabel, labels.EqualsOperator, []string{rc.Id().String()})
```

Comment:
ah that's right, before I forget... could you change the doc comments of `rcstore.Store.Create` accordingly? The comments say the pod labels are used, now they're not.